celery/platforms.py

```diff
@@ -229,6 +216,24 @@ def write_pid(self):
         finally:
             rfh.close()
 
+    def _is_stale(self):
+        """Check if the recorded pid in the pidfile is still alive."""
+        try:
+            pid = self.read_pid()
+        except ValueError as exc:
+            print('Broken pidfile found', file=sys.stderr)
+            return True
+        if not pid:
+            return True
+
+        try:
+            os.kill(pid, 0)
+        except os.error as exc:
+            if exc.errno == errno.ESRCH:
+                print('Stale pidfile exists', file=sys.stderr)
```

Comment:
I'd prefer to use the logging module here.
What do you think?

Reply:
Not sure but it is highly probable that logging has not been properly configured yet at this point, plus it is better not to rely on logging level configuration etc, since this is a deployment-related message.